theurgy.lic

```diff
@@ -391,7 +392,11 @@ class TheurgyActions
     return if @mat_unrolled
 
     if @prayer_mat
-      walk_to @prayer_mat_room['id'] if @prayer_mat_room
+      if @settings.theurgy_prayer_mat_room.is_a?(Hash)
+        walk_to @prayer_mat_room['id']
+      else @settings.theurgy_prayer_mat_room.is_a?(Integer)
```

Comment:
This should be `elsif`. As written, the conditional part is ignored.

```ruby
a = 1

if a == 2 then
	puts '2'
else a == 3
	puts '3'
end
```

Results in `3`.

-----

While you're editing this, can you add an `else` that outputs an error?

Reply:
Thank you! Have a look at the current iteration, it's tested, and I made the conditionals tighter, assuming they do what i think they do.